### DISCUSSION TITLE
Support multiple language

### DISCUSSION THREAD

**Discussion by hiren3897** [JamesBrill/react-speech-recognition#237] [Q&A] [answered]
Hello @JamesBrill,

Thanks for this best library

Does it support multiple language at the same time ? 

for example the user can speak in French or English or Spanish.

Currently, if i see the docs we can pass only one specific langauge in the `startListening`

is it already doable or needs some work?

**Comment by overkektus** [comment, marked as answer]
I think it's not possible. Because lib just use browser API's. And when you start listening, you should paste one language. So if you put for example 'en' as language and start to speak on French, then you'll get a mass of crap. It's api limitation. You could try deepgram for this. deepgram use AI for recognition. And of course they'll charge you for usage, but they give you a huge amount for test purposes. For serious projects, I think it's reasonable to use side services that will give you a huge benefit. Just try it if you're really interested in.
